src/openforms/conf/base.py

```diff
@@ -238,6 +239,7 @@
     "openforms.prefill",
     "openforms.prefill.contrib.demo",
     "openforms.prefill.contrib.eidas",
+    "openforms.prefill.contrib.communication_preferences",
```

Comment:
See my other comment about the naming of this prefill plugin module

Reply:
renamed the app to `customer_interactions`